docs/core/project-sdk/msbuild-props.md

```diff
@@ -16,99 +16,40 @@ This page is a reference for the MSBuild properties and items that you can use t
 - [ApiCompatRespectInternals](#apicompatrespectinternals)
-- [ApiCompatStrictMode](#apicompatstrictmode)
 - [ApiCompatSuppressionFile](#apicompatsuppressionfile)
 - [ApiCompatSuppressionOutputFile](#apicompatsuppressionoutputfile)
-- [ApiCompatValidateAssemblies](#apicompatvalidateassemblies)
-- [ApiCompatValidateAssembliesSemaphoreFile](#apicompatvalidateassembliessemaphorefile)
 - [NoWarn](#nowarn)
 - [RoslynAssembliesPath](#roslynassembliespath)
 
-### ApiCompatLeftAssemblies
-
-The `ApiCompatLeftAssemblies` item specifies the path to one or more assemblies that serve as the *left side* to compare. This property is required when `ApiCompatValidateAssemblies` is set to `true`.
-
-```xml
-<ItemGroup>
-  <ApiCompatLeftAssemblies Include="path/to/left-assembly" />
-</ItemGroup>
-```
-
-### ApiCompatLeftAssembliesReferences
-
-The `ApiCompatLeftAssembliesReferences` item specifies the path to assembly references or the underlying directories for the left side of the comparison.
-
-```xml
-<ItemGroup>
-  <ApiCompatLeftAssembliesReferences Include="path/to/assembly-references" />
-</ItemGroup>
-```
-
-### ApiCompatLeftAssembliesTransformationPattern
-
-The `ApiCompatLeftAssembliesTransformationPattern` item specifies a transformation pattern for the left side assemblies.
-
-```xml
-<ItemGroup>
-  <ApiCompatLeftAssembliesTransformationPattern Include="$(_ApiCompatCaptureGroupPattern)" ReplacementString="ref/$1/$2" />
-</ItemGroup>
-```
-
-### ApiCompatRightAssemblies
-
-The `ApiCompatRightAssemblies` item specifies the path to one or more assemblies that serve as the *right side* to compare. This property is required when `ApiCompatValidateAssemblies` is set to `true`.
-
-```xml
-<ItemGroup>
-  <ApiCompatRightAssemblies Include="path/to/right-assembly" />
-</ItemGroup>
-```
-
-### ApiCompatRightAssembliesReferences
-
-The `ApiCompatRightAssembliesReferences` item specifies the path to assembly references or the underlying directories for the right side of the comparison.
-
-```xml
-<ItemGroup>
-  <ApiCompatRightAssembliesReferences Include="path/to/assembly-references" />
-</ItemGroup>
-```
-
-### ApiCompatRightAssembliesTransformationPattern
-
-The `ApiCompatLeftAssembliesTransformationPattern` item specifies a transformation pattern for the right side assemblies.
-
-```xml
-<ItemGroup>
-  <ApiCompatRightAssembliesTransformationPattern Include="$(_ApiCompatCaptureGroupPattern)" ReplacementString="lib/$1/$2" />
-</ItemGroup>
-```
-
 ### ApiCompatStrictMode
 
-When set to `true`, the `ApiCompatStrictMode` property specifies that API compatibility checks should be performed in strict mode.
+When set to `true`, the `ApiCompatStrictMode` property specifies that API compatibility checks should be performed in *strict mode*.
 
 ```xml
 <PropertyGroup>
   <ApiCompatStrictMode>true</ApiCompatStrictMode>
 </PropertyGroup>
 ```
 
+For more information about strict mode, see [EnableStrictModeForBaselineValidation](#enablestrictmodeforbaselinevalidation).
```

Comment:
This links to package validation baseline validation strict mode. Can we have a dedicated page under i.e. fundamentals/apicompat/strictmode.md that explains what strict mode does in general and link to it?